package.json

```diff
@@ -104,6 +104,7 @@
     "ts-node": "~10.9.1",
     "tslint": "~6.1.3",
     "typescript": "~4.8.4",
-    "webpack-cli": "^4.10.0"
+    "webpack-cli": "^4.10.0",
+    "lodash": "^4.17.21"
```

Comment:
We don't want to add another 70KB unless it's absolutely necessary. forEach and includes could be done without lodash.